inst/include/Rcpp/protection/Shield.h

```diff
@@ -25,12 +25,16 @@ namespace Rcpp{
         return x ;
     }
 
+    inline SEXP Rcpp_unprotect(int i){
```

Comment:
This function wants to be `void` or else `g++` yells at you.

Reply:
You can cherry-pick it from 0df99c8f247f4af9a0e8b7d362086056e435b08a

Reply:
Thanks, done.